submissions/slack-status/src/slack.js

```diff
@@ -0,0 +1,50 @@
+export async function getSlackTeams() {
+    const slackTab = await chrome.tabs.create({
+        url: "https://app.slack.com/404",
+        active: false,
+    });
+    if (!slackTab.id) throw new Error("Failed to create tab");
+    console.debug(`Created Slack tab: ${slackTab.id}`);
+
+    const results = await chrome.scripting.executeScript({
+        target: { tabId: slackTab.id },
+        func: () => localStorage.getItem("localConfig_v2"),
+        injectImmediately: true,
+    });
+
+    const localConfig = results[0].result;
+    if (!localConfig) throw new Error("Failed to get xoxc");
+    const config = JSON.parse(localConfig);
+
+    setTimeout(() => {
```

Comment:
why wait before closing the tab here?

Reply:
This is due to a Chrome bug I’m afraid, as it sometimes errors out for seemingly no reason. The suggested fix on StackOverflow was to just use a timeout :/